cvss/parser.py

```diff
@@ -0,0 +1,27 @@
+import re
+
+from cvss import CVSS3, CVSS2
+from cvss.exceptions import CVSSError
+
+
+def parse_cvss_from_text(text):
+    # Vector may start with 'CVSS:3.0' substring - which is not extracted.
+    # Vector starts and ends with capitals
+    # which makes parsing from incorrectly spaced words easier.
+    matches = re.compile(r'((?:CVSS:3\.0/)?[A-Z][A-Za-z:/]+[A-Z])').findall(text)
```

Comment:
Maybe, just maybe, it would be better to match for cca `[A-Za-z:/.30]{26,}` may work basically the same.

Reply:
I find that `[A-Za-z:/.30]{26,}` does not match some useful cases like dot following the vector at the end of sentence.

I propose using `(?:CVSS:3\.0/)?[A-Za-z:/]{26,}` which accommodates dot at the end of sentence and also cases where a space is missing after the end of sentence. 
In comparison to original regex it does not support `vector + 'xxx'` cases where a vector is concatenated with another word.